cdci_data_analysis/config_dir/conf_env.yml.example

```
@@ -31,6 +31,9 @@ dispatcher:
     # maximum interval allowed during token refreshing
     token_max_refresh_interval: 604800
 
+    # timeout to re-submit the request
+    resubmit_timeout: 1800
```

Just that this makes sense to me

```suggestion
    resubmit_timeout: 900
```